### PR TITLE
ac_airplane OK and documented

### PR DESCRIPTION
**Pull request by inspectredc**

### CODE REVIEW

File: src/overlays/actors/ovl_Airplane/ac_airplane.c [hunk 1]
```diff
@@ -2,64 +2,334 @@
...
+        Matrix_scale(50.0f, 50.0f, 50.0f, 1);
+        gSPMatrix(POLY_OPA_DISP++, _Matrix_to_Mtx_new(game_play->state.gfxCtx),
+                  G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        gSPDisplayList(POLY_OPA_DISP++, glider_model);
+        CLOSE_DISPS(game_play->state.gfxCtx);
+    }
+}
+
+void aAp_RubberMove(f32* pValue, f32 target, f32 step) {
+    *pValue = *pValue + ((target - *pValue) * step);
```
Comment (Prakxo) [inline src/overlays/actors/ovl_Airplane/ac_airplane.c:77]:
```suggestion
    *pValue += (target - *pValue) * step;
```

File: src/overlays/actors/ovl_Airplane/ac_airplane.c [hunk 2]
```diff
@@ -2,64 +2,334 @@
...
+    if (this->state == Ap_STATE_FREE_FLY_MOVE) {
+        aAp_RubberMove(&this->zRot, 0.0f, 0.05f);
+    } else if (this->yRot < this->startYRot) {
+        aAp_RubberMove(&this->zRot, 50.0f, 0.05f);
+    } else {
+        aAp_RubberMove(&this->zRot, -50.0f, 0.05f);
+    }
+}
+
+static xyz_t wind1 = { 2.0f, 1.0f, 4.0f };
```
Comment (Prakxo) [inline src/overlays/actors/ovl_Airplane/ac_airplane.c:232]: Hmm non in function data has no static qualifier, and this symbol name doesn't repeat in the maps either

**Review by emilybrooks — APPROVED** [review]
nice work!

File: src/overlays/actors/ovl_Airplane/ac_airplane.c (hunk 1 above)
Comment (inspectredc) [reply src/overlays/actors/ovl_Airplane/ac_airplane.c:77]: this doesnt match :(

File: src/overlays/actors/ovl_Airplane/ac_airplane.c (hunk 2 above)
Comment (inspectredc) [reply src/overlays/actors/ovl_Airplane/ac_airplane.c:232]: ah ok will change!

**Pull request merged**